src/components/TodoAppContent/TodoAppContent.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { Todo } from '../../types/Todo';
+import { getTodos } from '../../api/todos';
+import { useAuth } from '../../Context/Context';
+import { TodoItem } from '../TodoItem';
+import { Filter } from '../Filter';
+import { filteredData } from '../../helper';
+import { FilterBy, ErrorMessage } from '../../types/types';
+import { Error } from '../Error';
+
+export const TodoAppContent: React.FC = () => {
+  const userId = useAuth();
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessage | null>(null);
+  const currentTodos = filteredData<Todo>(todos, filterBy);
+
+  useEffect(() => {
+    if (userId) {
+      getTodos(userId)
+        .then(setTodos)
+        .catch(() => setErrorMessage(ErrorMessage.Load));
+    }
+  }, []);
```

Comment:
React Hook useEffect has a missing dependency: 'userId'. Either include it or remove the dependency array